src/Components/Form/constant.ts

```diff
@@ -0,0 +1,3 @@
+export const REGEXP_PASSWORD =
+  /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%*#?&])[A-Za-z\d@$!%*#?&]{8,}$/;
+export const TEXT__REGEXP = new RegExp(/[A-Za-z]{3,}/);
```

Comment:
redundant constructor for literal
```suggestion
export const TEXT__REGEXP = /[A-Za-z]{3,}/;
```